shared/modules/rpc.utils.js

```diff
@@ -0,0 +1,54 @@
+import getFetchWithTimeout from './fetch-with-timeout';
+
+const fetchWithTimeout = getFetchWithTimeout(30000);
+
+/**
+ * Makes a JSON RPC request to the given URL, with the given RPC method and params.
+ *
+ * @param {string} rpcUrl - The RPC endpoint URL to target.
+ * @param {string} rpcMethod - The RPC method to request.
+ * @param {Array<unknown>} [rpcParams] - The RPC method params.
+ * @returns {Promise<unknown|undefined>} Returns the result of the RPC method call,
+ * or throws an error in case of failure.
+ */
```

Comment:
Review note: this is the same as the `jsonRpcRequest` that was removed from `ui/app/helpers/utils/util.js`, the code is unchanged